bigtable/tests/instance_admin_integration_test.cc

```diff
@@ -62,6 +74,34 @@ TEST_F(InstanceAdminIntegrationTest, ListInstancesTest) {
   }
 }
 
+/// @test Verify that InstanceAdmin::DeleteInstances works as expected.
+TEST_F(InstanceAdminIntegrationTest, DeleteInstancesTest) {
+  // The emulator does not support instance operations.
+  if (UsingCloudBigtableEmulator()) {
+    return;
+  }
+  std::string instance_id = "DeleteInstancesTest_instance";
+  bigtable::InstanceId instance(instance_id);
+  bigtable::DisplayName diplay_name(instance_id);
+  EXPECT_FALSE(
+      IsInstancePresent(instance_admin_->ListInstances(), instance_id));
+
+  std::vector<std::pair<std::string, bigtable::ClusterConfig>> clusters;
+  clusters.push_back(std::make_pair(
+      "sample-cluster",
+      bigtable::ClusterConfig(
+          "sample-cluster", 1234,
+          google::bigtable::admin::v2::StorageType::STORAGE_TYPE_UNSPECIFIED)));
```

Comment:
I think you have to set the storage type (use HDD) or the server will fail the request.

Reply:
changed it HDD.